langchain/src/schema/index.ts

```diff
@@ -101,8 +101,19 @@ export interface StoredGeneration {
 
 export type MessageType = "human" | "ai" | "generic" | "system" | "function";
 
+export type MessageContent =
+  | string
+  | (
+      | {
```

Comment:
i haven't written typescript -- so may have messed up the syntax... should this be something like:

{
          type: "text";
          text?: string;
} | {
          type: "image_url"; 
         image_url: string | { url: string };
}

Reply:
Fixed